extensions/src/ACRE2Arma/signal/signal.hpp

```diff
@@ -61,8 +65,12 @@ namespace acre {
         protected:
             uint32_t _debug_id;
             acre::signal::map_p _map;
-            acre::signal::model::multipath _signalProcessor_multipath;
-            acre::signal::model::Arcade    _signalProcessor_arcade;
+
+            acre::signal::model::Arcade      _signalProcessor_arcade;
```

Comment:
Arcade the only one capitalized?

Reply:
I will correct this in another PR (and rename it to casual). Let me have some feedback. Nevertheless, what convention do we take? Class names capital?

Reply:
Coding guidelines don't specify exactly, but use `snake_case` in all examples, so I'd say non-capitalized.